docs/csharp/tutorials/microservices.md

```diff
@@ -63,7 +63,9 @@ referenced above contains sections to each of the available installations.
 
 You'll also need to install a number of command line tools that support
 ASP.NET core development. The command line templates use Yeoman, Bower,
-Grunt, and Gulp. You may already have many of these tools, but if not,
+Grunt, and Gulp to be installed using the node.js's package manager npm which 
+can be installed from [nodejs.org](https://nodejs.org). 
+One may already have many of these tools, but if not,
```

Comment:
[The style guide](https://github.com/dotnet/docs/blob/master/styleguide/voice-tone.md) says that "you" should be used, not "one", so I think this should be changed back.